client/src/components/body/labspage/LabsPage.js

```diff
@@ -0,0 +1,386 @@
+                    }}
+                  >
+                    All Labs
+                  </button>
+                  <button
+                    className=" tw-bg-white tw-font-poppins tw-px-3 tw-py-3 tw-font-semibold tw-rounded-md
+                      tw-border-0 tw-shadow-md focus:tw-bg-primary-yellow focus:tw-shadow-xl hover:tw-bg-primary-yellow"
+                    onClick={() => {
+                      handleSearchChange("AI_MACHINE_LEARNING");
+                    }}
+                  >
+                    AI/Machine Learning
+                  </button>
+                  <button
+                    className=" tw-bg-white tw-font-poppins tw-px-3 tw-py-3 tw-font-semibold tw-rounded-md
+                      tw-border-0 tw-shadow-md focus:tw-bg-primary-yellow focus:tw-shadow-xl hover:tw-bg-primary-yellow"
+                    onClick={() => {
+                      handleSearchChange("ACCESSIBILITY");
+                    }}
+                  >
+                    Accessibility
+                  </button>
+                  <button
+                    className=" tw-bg-white tw-font-poppins tw-px-3 tw-py-3 tw-font-semibold tw-rounded-md
+                      tw-border-0 tw-shadow-md focus:tw-bg-primary-yellow focus:tw-shadow-xl hover:tw-bg-primary-yellow"
+                    onClick={() => {
+                      handleSearchChange("DIFF1");
+                    }}
+                  >
+                    Difficulty 1
+                  </button>
+                  <button
+                    className=" tw-bg-white tw-font-poppins tw-px-3 tw-py-3 tw-font-semibold tw-rounded-md
+                      tw-border-0 tw-shadow-md focus:tw-bg-primary-yellow focus:tw-shadow-xl hover:tw-bg-primary-yellow"
+                    onClick={() => {
+                      handleSearchChange("DIFF2");
+                    }}
+                  >
+                    Difficulty 2
+                  </button>
+                  <button
+                    className=" tw-bg-white tw-font-poppins tw-px-3 tw-py-3 tw-font-semibold tw-rounded-md
+                      tw-border-0 tw-shadow-md focus:tw-bg-primary-yellow focus:tw-shadow-xl hover:tw-bg-primary-yellow"
+                    onClick={() => {
+                      handleSearchChange("DIFF3");
+                    }}
+                  >
+                    Difficulty 3
+                  </button>
+                </div>
+
+                <div className="md:lg:tw-flex tw-flex-col md:lg:tw-justify-center sm:tw-grid-cols-2 tw-flex-wrap">
+                  {Array.from(displayedLabs.entries()).map(
+                    ([category, labArray]) => (
+                      <div
+                        key={category}
+                        className="tw-flex tw-flex-col tw-mb-4"
+                      >
+                        <text className="tw-font-bold tw-font-calibri tw-text-xl tw-w-full tw-text-left">
+                          {category}
+                        </text>
```

Comment:
Could we add just a little bit of padding below the category name?

![Screenshot 2024-11-05 at 12 57 15 PM](https://github.com/user-attachments/assets/e0593eb0-d573-4a0c-8860-ac8dd5eda142)